_posts/2016-06-17-carolina-intro.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "Introducing DSSG Fellow Carolina Johnson"
+data: 2016-06-17
+author: Carolina Johnson
+---
+
+![Small picture of Carolina]({{site.url}}/assets/images/carolina-small.jpg)
+
```

Comment:
@karawoo's picture rendered nicely on the web-page, so I would try to emulate what she did: 

https://github.com/uwescience/DSSG2016/blame/gh-pages/_posts/2016-06-24-introducing-dssg-fellow-kara-woo.md#L8-L9

In particular, I think that the front page only gets the *first paragraph* of text, so removing the line-break between the image link and the text in the next paragraph will make it render both your photo and a bit of your text (I think...). 

cc: @mdrouhard - if you feel like it, you can change yours to this format as well.